### PR TITLE
Added toDegrees doc

### LINKED ISSUE #919: [Term Entry] Java Math Methods: `.toDegrees()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.toDegrees()` Java method.

- In the `terms/` folder, make a new folder called `to-degrees`
- Inside the `to-degrees` folder, create a new markdown file called `to-degrees.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by sotiriszogos**
### Description

Closes #919 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dhanunjay18** [comment]
Please assignt it to me

**Comment by Dusch4593** [comment]
@Dhanunjay18 You've already been assigned to some issues; go ahead and get started on the PRs for those. We look forward to reviewing them. 😄

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:1]: @sotiriszogos please review the content standards [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and the term entry template [here](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md) to properly add metadata to this entry. You can also look at the source of any of the existing term entries to get an idea of what we're looking for.

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Title: '.toDegrees()'
+Description: 'It returns the measurement of the angle x in degrees.'
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:3]:
```suggestion
Description: 'Returns the measurement in degrees of an angle given in radians.'
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+The **`Math.toDegrees()`** method is a built-in math function in java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:6]:
```suggestion
The **`.toDegrees()`** method is a built-in math function in Java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Some edge cases for the `.toDegrees()` method include the following:
+
+- If the argument is `NaN`, this method will return `NaN`.
+- If the argument is `zero`, this method will return `zero` with the same sign as the argument.
+- If the argument is `infinity`, this method will return `infinity` with the same sign as the argument.
+
+## Example
+
+The following example showcases the `.toDegrees()` method being applied to two variables:
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:24]:
```suggestion
The following example demonstrates the `.toDegrees()` method being applied to two variables:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sotiriszogos, I did a first review and left some comments to be addressed. Also you should edit the description of this entry to complete the checklist.

**Review by SSwiniarski — APPROVED** [review]
@sotiriszogos, looks good for a second review.

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+The **`.toDegrees()`** method is a built-in math function in Java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
+
+## Syntax
+
+```pseudo
+Math.toDegrees(double x)
+```
+
+The return type of the `.toDegrees()` method is a double. The data type of the `x` parameter is also a double.
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:24]: "double" is a keyword in Java. It should be surrounded with backticks like so: `double`.

**Review by HishamT — APPROVED** [review]
Looks good! One minor change requested.

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.toDegrees()`** method is a built-in math function in Java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:16]:
```suggestion
The **`.toDegrees()`** method converts an angle, measured in radians, to an approximately equivalent angle, measured in degrees.
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+Title: '.toDegrees()'
+Description: 'Returns the measurement in degrees of an angle in radians.'
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:3]:
```suggestion
Description: 'Returns an angle measurement converted from radians to degrees.'
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.toDegrees()`** method is a built-in math function in Java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
+
+## Syntax
+
+```pseudo
+Math.toDegrees(double x)
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:21]:
```suggestion
Math.toDegrees(angle)
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+The return type of the `.toDegrees()` method is a `double`. The data type of the `x` parameter is also a `double`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:24]:
```suggestion
The data type of the `angle` parameter and return value of the `.toDegrees()` method is a `double`.
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+```java
+public class TestDegrees {
+  public static void main(String args[]) {
+       double x = 0.5235987756;
+       System.out.println(Math.toDegrees(x));
+       double y = 1.0471975512;
+       System.out.println(Math.toDegrees(y));
+   }
+}
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:44]:
```suggestion
public class TestDegrees {
  public static void main(String args[]) {
    double x = 0.5235987756;
    System.out.println(Math.toDegrees(x));
    
    double y = 1.0471975512;
    System.out.println(Math.toDegrees(y));
  }
} 
```

File: content/java/concepts/math-methods/terms/toDegrees/toDegrees.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+This results in the following output:
+
+```java
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/toDegrees/toDegrees.md:49]:
```suggestion
```shell
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@sotiriszogos Had some comments/suggestions to offer, as well. After this feedback is implemented, we should be good to merge. 😄

**Comment by Dusch4593** [comment]
Hey @sotiriszogos just wanted to check in and see how things were going with you and my latest comments/suggestions. Happy to help and/or answer questions. And remember, today is the last day for this PR to count towards Hacktoberfest.

**Comment by sotiriszogos** [comment]
Added all required changes

**Review by Dusch4593 — APPROVED** [review]
@sotiriszogos LGTM! 🚀

**Comment by Dusch4593** [comment]
Congrats and thank you so much for contributing to Codecademy Docs, @sotiriszogos! 🚀 🥳

The entry should now be live: https://www.codecademy.com/resources/docs/java/math-methods/toDegrees

Currently, your profile is listed anonymously on the entry page. To have it displayed, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account. Let us know if you'd need any help with doing this:

Screen Shot 2022-10-31 at 1 50 34 PM

Looking forward to working with you on your next contribution! 😄

A few things to note for the next one:

- Make sure to make a **_new pull request branch_** for the changes you intend on making. We generally discourage contributors from pushing new changes from their `main` branch because this could cause merge conflicts with the base `main` branch for `Codecademy/docs`.
- When you create a PR, always make sure to the check off the boxes under "Checklist" so that you are assured that you have set up your PR correctly.

**Pull request merged**